05.wc/wc.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main(options)
+  options = { l: true, w: true, c: true } if options.empty?
+  paths = ARGV
+  wc_count_files = build_wc_counts(paths:)
```

Comment:
ここに限らずですが、自作メソッドでキーワード引数を多用しすぎてます。
原則はキーワード引数ではない普通の引数として定義した方がシンプルで良いです。

```suggestion
  wc_count_files = build_wc_counts(paths)
```

「普通の引数だと都合が悪い」と思った場合だけキーワード引数を選択しましょう。

左辺と右辺の名前が微妙に異なるパターンなのでそこも見直してほしいです。
[ユーザーを見つけたら……Aさん「自転車に入れます」Bさん「人間に入れます」Cさん「ユーザーに入れます」 \| FBC](https://bootcamp.fjord.jp/pages/readable-variable-name)